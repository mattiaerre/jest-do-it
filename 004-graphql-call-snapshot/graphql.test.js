const getRegistryInfo = require('./graphql');

test('pink-pineapple registry info', async () => {
  const data = await getRegistryInfo();
  expect(data).toMatchSnapshot();
});
