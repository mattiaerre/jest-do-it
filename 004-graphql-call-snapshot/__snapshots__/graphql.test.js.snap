// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`pink-pineapple registry info 1`] = `
Object {
  "registry": Object {
    "baseUrl": "https://pink-pineapple.herokuapp.com/",
    "type": "oc-registry",
    "version": "0.36.27",
  },
}
`;
